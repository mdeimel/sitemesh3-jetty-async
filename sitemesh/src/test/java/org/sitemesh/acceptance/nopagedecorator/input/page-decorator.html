<html>
  <head>
    <title>Hello</title>
  </head>
  <body>

    <sitemesh:write property='body'>
      Ordinarily, this wouldn't get written out<br/>
      but because we're accessing the decorator directly and it's not being decorated
      by anything else, we leave the default content in. This is to help development.
    </sitemesh:write>

  </body>
</html>