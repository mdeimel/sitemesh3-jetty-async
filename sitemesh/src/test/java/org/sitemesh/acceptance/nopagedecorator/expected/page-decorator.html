<html>
  <head>
    <title>Hello</title>
  </head>
  <body>

    Ordinarily, this wouldn't get written out<br/>
    but because we're accessing the decorator directly and it's not being decorated
    by anything else, we leave the default content in. This is to help development.

  </body>
</html>